lib/yapay_ex.ex

```diff
@@ -3,16 +3,32 @@ defmodule YapayEx do
   Documentation for `YapayEx`.
   """
 
+  use Application
+
+  alias Finch.Response
   alias YapayEx.{Order, Request}
 
+  def start(_type, _args) do
+    children = [
+      YapayEx.Request.child_spec()
```

Comment:
I personally think that the child spec should be here. Since it's more related to the supervisor than the request itself.

I also wouldn't create a separate function for it but that's a matter of taste, I would like to avoid the extra indirection since it's a simple tuple 😅

Reply:
How would you solve this? I'm pretty new in Elixir and sometimes I don't know some of the best practices.  Thanks for you cr!

Reply:
Instead of having the child spec defined in [`YapayEx.Request.child_spec/1`](https://github.com/paulo-silva/yapay_ex/pull/6/files#diff-e2b39521f022f92b78031d9adcd1ef6cR7) you could only have the tuple in the `children`definition:

```elixir
children = [
  {Finch, name: Yapay.Request},
  # other children
]
```

* I type in the Github commentary, probably it is badly formated.